openapi/openapi-client/src/main/openapi3/master_rest_v1.yaml

```diff
@@ -456,6 +456,44 @@ paths:
                 type: string
                 format: binary
 
+  /api/v1/logger:
```

Comment:
This PR is amazing, can you provide an API to list all logger names since we don't know the logger names?

Reply:
sure

Reply:
done
<img width="1439" alt="image" src="https://github.com/user-attachments/assets/2ffb4aa4-42f9-43c7-9fc2-13745fa585b8" />

Reply:
154 items for the UT
<img width="1443" alt="image" src="https://github.com/user-attachments/assets/58a23973-d1e3-490c-a990-6ad9e64fdd99" />

Reply:
Using `loggerContext.getConfiguration.getLoggers` instead of `loggerContext.getLoggers`.


<img width="1227" alt="image" src="https://github.com/user-attachments/assets/0f03ee4e-ceeb-4925-9ef1-c517c2ebd7b2" />

Reply:
`loggerContext.getConfiguration.getLoggers` is reasonable. 

It only return the configured logger, including in the config file and changed by `POST /api/v1/loggers`.
https://github.com/apache/celeborn/blob/680b072b5bea852e8cf7733f0ec4d8aea127c51f/common/src/test/resources/log4j2-test.xml#L31-L40

Reply:
done:
<img width="1255" alt="image" src="https://github.com/user-attachments/assets/5c399a73-9f53-43a8-b337-5a79621abea4" />
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/16dc9ede-01bb-4e38-80fe-acb044ae6cc7" />